### ISSUE TITLE
Editable cell is not editable when scrolling to focus after a tab

### ISSUE THREAD

**Issue by spbagshaw** [angular-ui/ui-grid#4812]
In the version 3.0.0-rc.21 of angular-ui-grid calls to scrollToFocus() to a cell in a grid with enableCellEditOnFocus set to true would move the cursor to that cell and it would be immediately editable. In version 3.0.7 this no longer works in certain cases (see below).

I call the scrollToFocus() after a navigate (tab) event. It checks the values of certain cells and will jump the user to the next line based on a certain condition. However, if the cell that the user would have tabbed to without the scrollToFocus() is editable, then I think it sends an endEdit event and the cell the user has the focus on is no longer editable.

If the cell that the user would have tabbed to is not editable, the scrollToFocus() works fine.

Please see the Plunker which has instructions on how to prove the issue. http://plnkr.co/edit/bV9CYdHEEksvRtFUglhr?p=preview

Basically, this line shows the issue

{ name: 'name2', displayName: 'Name2 (editable)', width: '20%', enableCellEdit: true },

If the line is like below, all works fine

{ name: 'name2', displayName: 'Name2 (editable)', width: '20%', enableCellEdit: false },

**Comment by imbalind** [comment]
I managed a similar (but different) situation by manually raising afterCellEdit and beginCellEdit properly.

Anyhow, being this a heavily customized process, I'm closing this issue.

If you cannot solve this, in an effort to better manage this project and provide support to everyone we are directing "How-To" questions to Stack Overflow.

Our team and other users are making a concerted effort to respond people on SO.

Be sure to tag your question with [angular-ui-grid]!

**Comment by spbagshaw** [comment]
Can you please put your answer on Stack Overflow? I've had no answer there in about 2 weeks. 

http://stackoverflow.com/questions/33991169/scrolltofocus-to-cell-with-enablecelleditonfocus-true-does-not-make-the-cell-edi 

Still, I disagree that this is not a bug. It worked in a previous version of ui-grid, and it fails to work now depending on a setting in an adjacent column. Sure, there is a workaround (and I have my own involving 1 pixel wide spacer columns all over the place), but it seems hacky.
